### PR TITLE
Cera

### PR DESCRIPTION
**Pull request by C3ra906**
Probably don't want to merge mine since Ariel's PR hasn't been applied yet and I'm having a bunch of issues but here's what I have so far:

I copied and pasted the logic for winning/tying into a separate file called results.js for now to be used for backend. 

I have the splash.html set up and Splash.js. I can't figure out how to render Splash.js into splash.html, so I currently have it rendering through App.js along with the Banner and the Board just so I can see how it looks like on the page. I changed the index.js server file so it displays "Player wants to play" when Play button is clicked for now. 

I'm going to go watch some tutorials or read up on react and work on this more later today. Feel free to change anything.

### CODE REVIEW

File: src/components/results.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+      //socket.emit('move', { boardState: newBoard });
+      //setTurn(turn === 0 ? 1 : 0);
+    }
+  } else {
+    console.log(`Player ${curr} won!`);
+    // end game here
+  }
+
+  return winner;
+}
```
Comment (sashetov) [inline src/components/results.js:104]: i think its ok to merge but Ill move this code to the backend

**Review by sashetov — APPROVED** [review]

**Review by sashetov — CHANGES_REQUESTED** [review]
actually, on second pass, I think you should definitely not show the tic tac toe board ( hide it ) until the game has actually started. should be an easy fix

**Comment by sashetov** [comment]
also maybe use an asynchronous request instead of expecting a redirect and then change state based on the response you get

**Comment by C3ra906** [comment]
I just got the App.js to either display the splash content or the board. With the onclick event, I wasn't sure how to control the Splash.js from App.js...so I just copied the content of Splash.js into App.js. Splash.js and Splash.html are currently redundant. I'm going to look into it more tomorrow to see if I can separate out Splash from App. Let me know if you know how to link onclick event from one js file to another in React!

**Comment by gleason9113** [comment]
I think that, after looking at some other code examples, the idea is in fact to render different components within the same index.html.  I'm okay to merge this whenever you want.  The merge conflicts are the result of merging Alex's branch- the conflicts are all in the server file and should be resolved in favor of main.

**Pull request merged**